src/Raven.Client/ServerWide/Commands/TcpConnectionInfo.cs

```diff
@@ -7,15 +7,25 @@ public class TcpConnectionInfo
         public int Port;
         public string Url;
         public string Certificate;
-
+        public string[] TcpServerUrls;
         public DynamicJsonValue ToJson()
         {
-            return new DynamicJsonValue
+            var res =  new DynamicJsonValue
             {
                 [nameof(Port)] = Port,
                 [nameof(Url)] = Url,
                 [nameof(Certificate)] = Certificate
             };
+
+            var array = new DynamicJsonArray();
+            foreach (var url in TcpServerUrls)
```

Comment:
Don't we need to handle a null here?

Reply:
No, because we always populate the server list and if it is not set it will contain the ServerTcpUrl as a sole entry

Reply:
What happens if you are reading from a 4.1 or a 4.0 server?